Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
I think we can keep the `https` versioned. If someone is having problems with the `ssl` locally, just change to install/update the gems, but doesn't commit.